<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>280</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QComboBox" name="cbxArea">
      <item>
       <property name="text">
        <string>宜蘭縣</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>桃園市</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>新竹縣</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>苗栗縣</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>彰化縣</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>南投縣</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>雲林縣</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>嘉義縣</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>屏東縣</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>臺東縣</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>花蓮縣</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>澎湖縣</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>基隆市</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>新竹市</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>嘉義市</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>臺北市</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>高雄市</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>新北市</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>臺中市</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>臺南市</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>連江縣</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>金門縣</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cbxLocation"/>
    </item>
    <item>
     <widget class="QSpinBox" name="spxTime">
      <property name="readOnly">
       <bool>false</bool>
      </property>
      <property name="maximum">
       <number>24</number>
      </property>
      <property name="singleStep">
       <number>3</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnEnter">
      <property name="text">
       <string>Query</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>181</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelWeather">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>171</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
